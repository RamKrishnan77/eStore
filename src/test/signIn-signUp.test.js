import {shallow} from 'enzyme'
import React from 'react'
import SignIn from '../components/sign-in.component'
import SignUp from '../components/sign-up.component'

//Snapshot testing of SignIn using Eznyme’s Full DOM rendering 

it('expect to render SignIn component', () => {
    expect(shallow(<SignIn></SignIn>)).toMatchSnapshot()
})

//Snapshot testing of SignUp using Eznyme’s Full DOM rendering 

it('expect to render SignUp', () => {
    expect(shallow(<SignUp></SignUp>)).toMatchSnapshot()
})

