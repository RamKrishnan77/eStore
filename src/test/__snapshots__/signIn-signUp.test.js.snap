// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SignIn component 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`expect to render SignUp 1`] = `
<div
  className="sign-up"
>
  <h2
    className="title"
  >
    I do not have a account
  </h2>
  <span>
    Sign up with your email and password
  </span>
  <form
    className="sign-up-form"
    onSubmit={[Function]}
  >
    <FormInput
      label="Display Name"
      name="displayName"
      onChange={[Function]}
      required={true}
      type="text"
      value=""
    />
    <FormInput
      label="Email"
      name="email"
      onChange={[Function]}
      required={true}
      type="email"
      value=""
    />
    <FormInput
      label="Password"
      name="password"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <FormInput
      label="Confirm Password"
      name="confirmPassword"
      onChange={[Function]}
      required={true}
      type="password"
      value=""
    />
    <CustomButton
      type="submit"
    >
      SIGN UP
    </CustomButton>
  </form>
</div>
`;
